kolibri_explore_plugin/assets/package.json

```diff
@@ -7,7 +7,8 @@
     "bootstrap": "^4.5.3",
     "bootstrap-vue": "^2.21.2",
     "eos-components": "^0.1.0",
-    "kolibri-design-system": "https://github.com/learningequality/kolibri-design-system"
+    "kolibri-design-system": "https://github.com/learningequality/kolibri-design-system",
+    "underscore": "^1.13.1"
```

Comment:
:+1:

Reply:
In template_ui we have "underscore": "^1.11.0", so we need to bump that one to keep our dependencies the same. I'll do as follow-up. Also lerna has commands to manage the same dependency in multiple packages.